app/app.go

```diff
@@ -341,10 +343,10 @@ func New(
 		packetforwardkeeper.DefaultRefundTransferPacketTimeoutTimestamp,  // refund timeout
 	)
 	// PacketForwardMiddleware is used only for version 2.
-	transferStack = module.NewVersionedIBCModule(packetForwardMiddleware, transferStack, v2, v2)
+	transferStack = module.NewVersionedIBCModule(packetForwardMiddleware, transferStack, v2, v3)
```

Comment:
should we have some kind of test to avoid forgetting to bump versioned ibc modules?